client/transport/rest/src/main/java/io/a2a/client/transport/rest/RestTransportConfig.java

```diff
@@ -0,0 +1,17 @@
+package io.a2a.client.transport.rest;
+
+import io.a2a.client.http.A2AHttpClient;
+import io.a2a.client.transport.spi.ClientTransportConfig;
+
+public class RestTransportConfig extends ClientTransportConfig<RestTransport>  {
```

Comment:
The main README should also be updated to add HTTP+JSON/REST configuration instructions to the server and client sections.